docs/source/guide/ml.md

```diff
@@ -145,6 +170,44 @@ If you want to retrieve predictions manually for a list of tasks **using only an
 }
 ```
 
+Your Machine Learning prediction method signature:
+
+```python
+def predict(self, tasks, **kwargs)
+```
+
+The **tasks** param - list of task for prediction, can be a list or a single task in a list. Tasks format is common for Label Studio, check format in [guide](/guide/export.html#Label-Studio-JSON-format-of-annotated-tasks).
```

Comment:
The **tasks** param - list of tasks for prediction, can be a list or a single task in a list. Tasks format is common for Label Studio, check the format in [guide](/guide/export.html#Label-Studio-JSON-format-of-annotated-tasks).

Reply:
Fixed